### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by avinoth**
Discussed at #44, Added a new pull request template.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Please review our [Contribution Guidelines](https://github.com/avinoth/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.
```
Comment (doomspork) [inline .github/PULL_REQUEST_TEMPLATE.MD]: This file is hardcoded to your repo.

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Make sure your changes ticks off the below checklist.
+
+- [ ] Company Name
```
Comment (doomspork) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Couple thoughts on this list:
- For consistency let's only capitalize the first word like you did in the last 2.  
- We should add something about their GitHub link
- Do we need the last one?  I'd hope that is a given.

What about something like:

- [ ] Company name
- [ ] Company description
- [ ] GitHub link
- [ ] How Elixir is used
- [ ] Sorted alphabetically

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Please review our [Contribution Guidelines](https://github.com/avinoth/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.
+
+Make sure your changes ticks off the below checklist.
```
Comment (doomspork) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's tell them what to do:

"Review your changes and complete the checklist below:"

**Comment by doomspork** [comment]
This is great, thanks @avinoth.  I left some feedback for you.

**Comment by avinoth** [comment]
@doomspork It's so embarrassing with the mistakes. I've corrected it, take a look at it when you have a chance.

Oh, and also please squash and merge to make sure the last commit message doesn't go into history :blush:

**Comment by doomspork** [comment]
Good morning @avinoth!  No need to be embarrassed, I _really_ appreciate you taking the initiative to do this.

This looks good to me, thank you again :+1:

**Pull request merged**